### DISCUSSION TITLE
More auto-approve options?

### DISCUSSION THREAD

**Discussion by narthur** [chdsbd/kodiak#895] [Ideas]
Kodiak looks super cool. Wondering--it would be really neat if you could configure auto-approval to happen if, for example, the pr was opened by any collaborator, and also all checks, including optional checks, passed. That way a human review could be required only in situations where it's likely needed.

I'm thinking this would be nice in cases where you want to keep an eye on certain changes or metrics, but don't want to entirely exclude them with a required check. For example, if test coverage goes down, or bundle size goes up, you could have Kodiak disqualify the PR for auto-approval even if those things might be ok from time to time as long as it's been sanity-checked.

Is this already possible, or could it be added in the future?

**Comment by chdsbd** [comment]
This isn't possible at the moment, but Kodiak could be updated to support more checks to conditionally merge PRs.
